src/components/address-output/_macro.njk

```diff
@@ -0,0 +1,18 @@
+{% macro onsAddressOutput(params) %}
+    <div class="address-output{% if params.classes is defined and params.classes %} {{ params.classes }}{% endif %}">
+        <p class="address-output__lines">
+            {% if params.line1 is defined and params.line1 %}
```

Comment:
I think we could do with adding 'unit' and 'organisation' address lines to cater for CE and business addresses